### ISSUE TITLE
Elemental Pull Causing Enemy Hostility

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#608]
This ability seems to, semi-randomly, do damage to enemies and make them hostile, even overriding Simple Offensive Suppression mod.

---

## Game Version
1.7.36.0

## Language
English (EN)

## New Game
Yes

## Reproduction Steps
Use Elemental Pull starborn power in a variety of situations. Sometimes it will get you attacked or arrested, other times its fine.

**Comment by Constellation-VASCO** [comment]
Any time I've encountered this, it usually seems to have been because of an item that would be considered stolen getting moved. It seems that, if you directly cause any such item to be moved at all, in any way, it aggros the NPCs immediately. You can even repro this by holding E to pick up and move an item without actually taking it, and the NPCs will immediately aggro as if you stole it.

**Comment by Constellation-VASCO** [comment]
Ahhh that's such an interesting interaction.

**Comment by Constellation-VASCO** [comment]
It was kind of funny moving a beach ball and getting lit up at Paradiso

**Comment by Constellation-VASCO** [comment]
Ha yeah.

My first encounter with that unexpected behavior was at the clinic, when I moved a chunks apple or some other similar food item that was in my way to see something I wanted to see.

I moved it without taking it, and instantly the turrets, security, and even the robots from the storage area came and attempted to dole out some swift justice. I went ahead and fought them all off just to teach them a lesson and some cathartic release for dealing with silly behaviors like that. Then I reloaded from the autosave upon entering the station and whoever bumped that apple or whatever caused it to be in the way apparently didn't do it, that time, so I was able to go about my business in peace. 😅
